app/src/main/java/com/globant/example/mentorapp/mvp/base/BaseRecyclerViewAdapter.java

```diff
@@ -0,0 +1,16 @@
+package com.globant.example.mentorapp.mvp.base;
+
+/**
+ * Generic class to define basic RecycleView behavior
+ * Created by oscar.amezquita on 21/07/2017.
+ */
+
+public class BaseRecyclerViewAdapter {
+
+    /**
+     * Listener to handle selected User click.
+     */
+    public interface onUserClick {
```

Comment:
Do you expect to have user clicks all your adapters? This listener can be moved to the fragment that implements the adapter